src/native/corehost/apphost/apphost.windows.cpp

```diff
@@ -323,6 +323,20 @@ namespace
 
         trace::verbose(_X("Showing error dialog for application: '%s' - error code: 0x%x - url: '%s' - details: %s"), executable_name, error_code, url.c_str(), details.c_str());
 
+        HMODULE user32 = ::LoadLibraryExW(L"user32.dll", nullptr, LOAD_LIBRARY_SEARCH_SYSTEM32);
+        if (user32 != nullptr)
+        {
+            typedef DPI_AWARENESS_CONTEXT(WINAPI* set_thread_dpi)(DPI_AWARENESS_CONTEXT context);
```

Comment:
```suggestion
            using set_thread_dpi = DPI_AWARENESS_CONTEXT(WINAPI*)(DPI_AWARENESS_CONTEXT context);
```

Nit on using newer C++ syntax.